build/index.ts

```diff
@@ -474,6 +474,11 @@ export async function buildDocument(
   doc.mdn_url = document.url;
   doc.locale = metadata.locale as string;
   doc.native = LANGUAGES.get(doc.locale.toLowerCase())?.native;
+  const browserCompat = metadata["browser-compat"];
+  doc.browserCompat =
+    browserCompat && Array.isArray(browserCompat)
+      ? browserCompat
+      : [browserCompat];
```

Comment:
This is ending up as `[null]` for docs without a bcd table.

Struggling to come up with something better than 
```suggestion
  doc.browserCompat = browserCompat
    ? Array.isArray(browserCompat)
      ? browserCompat
      : [browserCompat]
    : undefined;
```